Meta/run.sh

```diff
@@ -76,9 +76,13 @@ else
     SERENITY_QEMU_DISPLAY_DEVICE="VGA,vgamem_mb=64 "
 fi
 
+if [ -z "$SERENITY_DISABLE_GDB_SOCKET" ]; then
```

Comment:
I haven't seen the crash mentioned in the commit message when running QEMU on windows (in WSL or the native windows QEMU). Is it possible you have a port conflict and something is already bound to the socket that GDB attempts to open on your machine?

Reply:
The crash only happens on the Windows build of QEMU, and it isn't only happening on my machine, see #5695, and also two other users in #build-problems on Discord.